tests/unit/test_refresh_task_critical.py

```
@@ -0,0 +1,376 @@
+        with (
+            patch("refresh_task._get_mp_context", return_value=ctx),
+            patch.dict(os.environ, {"INKYPI_PLUGIN_ISOLATION": "process"}),
+        ):
+            try:
+                task._execute_with_policy(
+                    self._mock_action(),
+                    {"id": "test"},
+                    datetime.now(UTC),
+                    "req-1",
+                )
+                assert False, "Expected RuntimeError"
```

Ruff (0.15.7)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)